packages/blocks/src/root-block/edgeless/edgeless-keyboard.ts

```diff
@@ -219,9 +246,55 @@ export class EdgelessPageKeyboardManager extends PageKeyboardManager {
         global: true,
       }
     );
+
     this._bindToggleHand();
   }
 
+  /**
+   * The Event Dispatcher will only listen to keyboard events that bubble up to the host element,
+   * which means that when we do not have focus on the editor, we can only listen to keyboard events on the document.
+   * That's why we need this function to bind hotkeys that need to be triggered even when the editor does not have focus.
+   *
+   * To support multiple editors, we determine whether to activate the event callback based on the user's mouse position.
+   */
+  private _edgelessHotKeyActive = false;
+  bindEdgelessHotKey(keymap: Record<string, UIEventHandler>) {
```

Comment:
This seems not right. We shouldn't add specific implementation for edgeless mode of a common feature.